src/app/config/confparse.h

```diff
@@ -57,8 +55,8 @@ typedef enum config_type_t {
 typedef union {
   char **STRING;
   char **FILENAME;
-  int *UINT; /* yes, really: Even though the confparse type is called
-              * "UINT", it still uses the C int type -- it just enforces that
+  int *POSINT; /* yes, really: Even though the confparse type is called
+              * "POSINT", it still uses the C int type -- it just enforces that
```

Comment:
I'm happy to clean this comment up, but I'd rather do so as part of #30864, since that branch already moves this declaration around.

How about "This is indeed an int, not a unsigned int. For historical reasons, many configuration values are restricted to the range [0,INT_MAX]." ?